source/base/ByteBuffer.ooc

```diff
@@ -28,26 +28,29 @@ ByteBuffer: class {
 	size ::= this _size
 	_referenceCount: ReferenceCounter
 	referenceCount ::= this _referenceCount
-	init: func (=_pointer, =_size) {
+	_owner: Bool
```

Comment:
In `VectorList` it refers to each element in the list though, not the backend itself, as is the case here. Still, I guess it would work. Thanks for the suggestion.